contracts/test/accounts/Governor.sol

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.6.11;
+pragma experimental ABIEncoderV2;
+
+import { IERC20 } from "../../../modules/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+
+import { IMapleGlobals } from "../../interfaces/IMapleGlobals.sol"; 
+
+contract Governor {
+
+    /************************/
+    /*** Direct Functions ***/
+    /************************/
+    function setValidLoanFactory(address globals, address factory, bool valid)         external { IMapleGlobals(globals).setValidLoanFactory(factory, valid); }
+    function setValidPoolFactory(address globals, address factory, bool valid)         external { IMapleGlobals(globals).setValidPoolFactory(factory, valid); }
+    function setPriceOracle(address globals, address asset, address oracle)            external { IMapleGlobals(globals).setPriceOracle(asset, oracle); }
+    function setValidSubFactory(address globals, address fac, address sub, bool valid) external { IMapleGlobals(globals).setValidSubFactory(fac, sub, valid); }
+    function setCalc(address globals, address calc, bool valid)                        external { IMapleGlobals(globals).setCalc(calc, valid); }
```

Comment:
Are we really going to start doing these single line functions with no new lines between them now? I vote no, haha.